<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>602</width>
    <height>451</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>602</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="alg1Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="alg2Label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm 2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compareBtn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Comparar algoritmos selecionados</string>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QComboBox" name="vectorList">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ordenação do vetor a ser ordenado</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="vectorLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vector order:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="graphicGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="title">
     <string>Graphic</string>
    </property>
    <widget class="QRadioButton" name="timeRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="swapsRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Swaps</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="compareRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Comparissons</string>
     </property>
    </widget>
    <widget class="QCustomPlot" name="graphic" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>401</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="algList1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Algoritmo 1 a ser comparado</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="calcLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculations done:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="algList2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Algoritmo 1 a ser comparado</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <zorder>graphicGroup</zorder>
   <zorder>alg1Label</zorder>
   <zorder>alg2Label</zorder>
   <zorder>compareBtn</zorder>
   <zorder>vectorList</zorder>
   <zorder>vectorLabel</zorder>
   <zorder>algList1</zorder>
   <zorder>progressBar</zorder>
   <zorder>calcLabel</zorder>
   <zorder>algList2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="clearPlot"/>
    <addaction name="saveastxt"/>
    <addaction name="about_2"/>
    <addaction name="exitAction_2"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="exitAction">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="clearPlot">
   <property name="text">
    <string>Limpar Gráfico</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="saveastxt">
   <property name="text">
    <string>Salvar em Texto...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="exitAction_2">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="especs">
   <property name="text">
    <string>Especificações</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="let_esp">
   <property name="text">
    <string>LET Especial #0</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="about_2">
   <property name="text">
    <string>Sobre...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
